packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -96,19 +171,31 @@ export async function getManifestResponseAsync({
       key: mainModuleName,
       contentType: 'application/javascript',
       url: bundleUrl,
+      fileExtension: '.fake', // shouldn't need this but android crashes without it
```

Comment:
We intentionally omit fileExtension from the launchAsset. We should look into this because I'd expect it to work and further more, how were updates working at all before we introduced fileExtension? Maybe it's a difference between development and production updates?

Reply:
Looking into this